fuel-kotlinx-serialization/build.gradle.kts

```diff
@@ -1,10 +1,9 @@
 import org.jetbrains.kotlin.gradle.ExperimentalKotlinGradlePluginApi
 import org.jetbrains.kotlin.gradle.dsl.JvmTarget
-import org.jetbrains.kotlin.gradle.targets.js.dsl.ExperimentalWasmDsl
 
 plugins {
     kotlin("multiplatform")
-    kotlin("plugin.serialization") version "2.0.21"
+    kotlin("plugin.serialization") version "2.1.0"
```

Comment:
[nits] Do you think that we should move this version into toml file so we can control all of the versioning in one place? Not blocking this PR though.

Reply:
I can create a new PR for your suggestions.